lib/screen/account/name_persona_page.dart

```diff
@@ -90,7 +98,14 @@ class _NamePersonaPageState extends State<NamePersonaPage> {
                       AuTextField(
                           title: "",
                           placeholder: "enter_alias".tr(),
-                          controller: _nameController),
+                          controller: _nameController,
+                          onChanged: (valueChanged){
+                            if((_nameController.text == "" && isSaveAliasDisable == false)||
```

Comment:
User can inputs spaces, so we should trim the text before check empty/not empty.

Reply:
done